napari/components/_viewer_key_bindings.py

```diff
@@ -56,18 +55,21 @@ def toggle_ndisplay(viewer: Viewer):
 # ```
 # RuntimeError: wrapped C/C++ object of type CanvasBackendDesktop has been deleted
 # ```
-@register_viewer_action(trans._("Toggle theme."))
+@register_viewer_action(trans._("Toggle current viewer theme."))
 def toggle_theme(viewer: Viewer):
```

Comment:
```suggestion
def toggle_theme(viewer: ViewerModel):
```
As it is tested this way.